test_unbounded_knapsack.py

```diff
@@ -0,0 +1,49 @@
+"""
+Created on Tue Aug 31 16:31:07 2022
+
+This file contains the test-suite for the unbounded_knapsack problem.
+"""
+import unittest
+
+from unbounded_knapsack import unbounded_knapsack as k
+
+
+class Test(unittest.TestCase):
+    def test_base_case(self):
+        """
+        test for the base case
+        """
+        capacity = 0
+        value = [0]
+        weight = [0]
+        length = len(val)
+        self.assertEqual(k.unbounded_knapsack(capacity, weight, value, length), 0)

```

Comment:
We can simplify to...
```suggestion
        self.assertEqual(k.unbounded_knapsack(capacity=0, weight=[0], value=[0], length=1), 0)
```